app/components/views/Bond/index.tsx

```diff
@@ -749,10 +754,39 @@ export const Bond: FC<Props> = (props) => {
                 <Spinner />
               </div>
             ) : (
-              <ButtonPrimary
-                {...getButtonProps()}
-                className={styles.submitButton}
-              />
+              <Box
+                sx={{ display: "flex", flexDirection: "column", width: "100%" }}
+              >
+                <ButtonPrimary
+                  {...getButtonProps()}
+                  className={styles.submitButton}
+                />
+                {view === "redeem" && !showSpinner && (
+                  <Box
+                    sx={{
+                      display: "flex",
+                      width: "100%",
+                      alignItems: "center",
+                      justifyContent: "center",
+                      paddingRight: "16px",
+                    }}
+                  >
+                    <Checkbox
+                      checked={shouldAutostake}
+                      onChange={handleAutostakeCheck}
+                      sx={{ color: "#fff" }}
```

Comment:
we will be adding lightmode/darkmode to the dapp soon, so always use color variables, in this case I think it's `--font-01`
https://github.com/KlimaDAO/klimadao/blob/c15c6fcf288d5bc6eafad69316ccda6f727da40a/lib/theme/variables.css#L34

Reply:
updated, I will keep this in mind for the future.